mlflow/langchain/__init__.py

```diff
@@ -94,6 +98,19 @@ def get_default_conda_env():
     return _mlflow_conda_env(additional_pip_deps=get_default_pip_requirements())
 
 
+def get_retriever_pyfunc_model(fn):
```

Comment:
Nit: Can we make this internal by prefixing with underscore so that it doesn't show up in the docs?

```suggestion
def _get_retriever_pyfunc_model(fn):
```